packages/@n8n/benchmark/scripts/bootstrap.sh

```diff
@@ -43,7 +43,6 @@ sudo systemctl disable getty@tty1.service
 sudo systemctl disable serial-getty@ttyS0.service
 # Snap
 sudo systemctl disable snapd.service
-sudo apt remove snapd
```

Comment:
This was missing `-y`, but on second thought I don't think we need to remove it, disabling is enough